lgc/elfLinker/GlueShader.h

```diff
@@ -47,6 +47,8 @@ class GlueShader {
   // Create a fetch shader
   static GlueShader *createFetchShader(PipelineState *pipelineState, llvm::ArrayRef<VertexFetchInfo> fetches,
                                        const VsEntryRegInfo &vsEntryRegInfo);
+  // Create a color export shader
+  static GlueShader *createColorExportShader(PipelineState *pipelineState, llvm::ArrayRef<ColorExportInfo> exports);
```

Comment:
This should really return a `std::unique_ptr<GlueShader>`.

I can see that this is kind of pre-existing bad style here, but now would be the perfect time to do better :)

Reply:
done.